cyto_dl/models/im2im/utils/postprocessing/auto_thresh.py

```diff
@@ -23,7 +23,10 @@ def thresh_func(image):
         self.thresh_func = thresh_func
 
     def __call__(self, image):
-        image = image.detach().cpu().float().numpy()
+        # Only get numpy array from torch tensor if tensor is passed in
+        # Allows np.ndarray to be passed in directly
+        if not isinstance(image, np.ndarray):
+            image = image.detach().cpu().float().numpy()
```

Comment:
(feel free to ignore since this is out of the scope of the pr)
there's also a [detach]("\\allen\aics\assay-dev\users\Benji\CurrentProjects\im2im_dev\cyto-dl\logs\train\runs\diffae\orig_model\2024-12-06_12-47-05\checkpoints\last.ckpt") function that does the detach->cpu->numpy conversion, so this could be replaced with 
`img = detach(img).astype(float)`